includes/definitions/discovery/ray.yaml

```diff
@@ -37,7 +37,7 @@ modules:
             data:
                 -
                     oid: ber
-                    num_oid: '.1.3.6.1.4.1.33555.4.3.2.11.{{ $index }}'
+                    num_oid: '.1.3.6.1.4.1.33555.1.3.2.11.{{ $index }}'
```

Comment:
Looks better for the OID. But looking at the MIB, it seems that a divisor is necessary here:
`1.3.6.1.4.1.33555.1.3.2.11: Bit Error Rate (BER) multiplied by 10^9. `

Reply:
I don't think so. As far as I have seen graphs of other radio links, there was no divider used and the graphs showed the same values if there was a ber.

Reply:
The value is not divisible and is reported correctly.
https://www.techtarget.com/whatis/definition/bit-error-rate-BER

Reply:
All right. Then you just need to generate again the JSON file and you should be good to go.